### PR TITLE
Update the README file [coding party] git workshop #1, challenge #1

### LINKED ISSUE #40: [coding party] git workshop #1, challenge #1
Add instructions on how to fork the devopsloft/training repo in the README.md file.

The file to update:
training/git/1/1/README.md

Verify that GitHub renders the content properly

### PR DESCRIPTION
**Pull request by mosheeven**
fix #40

### CODE REVIEW

File: git/1/1/README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # git workshop #1, challenge #1
+Follow the following steps for fork devopsloft/training repo:
+1. Fork devopsloft/training to your git account.
```
Comment (lmilbaum) [inline git/1/1/README.md]: GitHub namespace

Comment (lmilbaum) [reply git/1/1/README.md]: switch every your_account with namespace

File: git/1/1/README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
 # git workshop #1, challenge #1
+Follow the following steps for fork devopsloft/training repo:
+1. Fork devopsloft/training to your git account. 
+2. Clone the repo from your own account to local computer. https://github.com/<your_account>/training.git
+3. Add upstream on your local computer. git remote add upstream https://github.com/devopsloft/training.git
```
Comment (lmilbaum) [inline git/1/1/README.md]: upstream rempote

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: git/1/1/README.md (hunk 1 above)
Comment (mosheeven) [reply git/1/1/README.md]: Fixed.

File: git/1/1/README.md (hunk 2 above)
Comment (mosheeven) [reply git/1/1/README.md]: Fixed.

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**